src/common/line_entry.rs

```diff
@@ -301,6 +301,16 @@ mod tests {
     }
 
     mod get_control_comment {
+        use crate::lint_kind::LintKind;
+        use std::str::FromStr;
+
+        fn as_lintkind_vec(vec: Vec<&str>) -> Vec<LintKind> {
```

Comment:
I don't think that we need a special function in the tests module to convert vectors.
Just use `vec![LintKind::UnorderedKey]`.